combinetf2/physicsmodels/helpers.py

```
@@ -0,0 +1,136 @@
+import importlib
+import re
+
+import tensorflow as tf
+
+# dictionary with class name and the corresponding filename where it is defined
+baseline_models = {
+    "Basemodel": "physicsmodel",
+    "Project": "project",
+    "Normalize": "project",
+    "Ratio": "ratio",
+    "Normratio": "ratio",
+}
+
+
+def instance_from_class(class_name, *args, **kwargs):
+    if "." in class_name:
+        # import from full relative or abslute path
+        parts = class_name.split(".")
+        module_name = ".".join(parts[:-1])
+        class_name = parts[-1]
+    else:
+        # import one of the baseline models
+        if class_name not in baseline_models:
+            raise ValueError(
+                f"Model {class_name} not found, available baseline models are {baseline_models.keys()}"
+            )
+        module_name = f"combinetf2.physicsmodels.{baseline_models[class_name]}"
+
+    # Try to import the module
+    module = importlib.import_module(module_name)
+
+    try:
+        cls = getattr(module, class_name)
+    except AttributeError:
```

Probably better to just crash here rather than skip the missing model silently